packages/components/src/context-tag/context-tag.tsx

```diff
@@ -51,7 +66,7 @@ const Label = ({
   type = 'default',
```

Comment:
We'll have to change this in case of renaming type from 'default' to 'user'

Reply:
But that is a diff component, Label, no? It's not even from DS, is it? And "default" in this case is used for more cases, not just "user".

Reply:
Sorry, you're right, I didn't see whole context of the file.